packages/EtendoHookBinder/src/etendo.d.ts

```diff
@@ -0,0 +1,104 @@
+export declare global {
+  export namespace Etendo {
+    type WindowId = string;
+    type ColumnId = string;
+
+    type Entity = 'Order' | 'Invoice' | 'Product' | 'OrderLine';
```

Comment:
I am not clear how it can be typed, probably String type, we do not know the entity names we can receive.